prometheus-xmpp-alerts

```diff
@@ -106,7 +106,7 @@ args = parser.parse_args()
 logging.basicConfig(level=args.loglevel, format='%(levelname)-8s %(message)s')
 
 with open(args.config_path) as f:
-    config = yaml.load(f)
+    config = yaml.load(f, Loader=yaml.FullLoader)
```

Comment:
Please keep backwards compatibility - older versions don't support the Loader argument.

Reply:
I've restored backwards-compatibility and path the code when building the image, as i wasn't able to just silence the warning with any variation of `PYTHONWARNINGS=ignore::yaml.YAMLLoadWarning`.